resources/core/templates/tests/test-external-solution.yaml

```diff
@@ -26,7 +26,7 @@ spec:
         - ip: {{ $.Values.global.minikubeIP }}
           hostnames:
             - "connector-service.{{ $.Values.global.ingress.domainName }}"
-            - "counter-service-{{ $val.testsuite }}.{{ $.Values.global.ingress.domainName }}"
+            - "ctr-svc-{{ $val.testsuite }}.{{ $.Values.global.ingress.domainName }}"
```

Comment:
What is the reason for changing this name? I feel like `ctr-svc` does not tell much for someone who is not familiar with the code.

Reply:
`counter-service` name were too long and sometimes the e2e tests `compass-runtime-agent-adapter-external-solution`